index.js

```diff
@@ -5,22 +5,22 @@ function getRandomInt(max) {
     return Math.floor(Math.random() * Math.floor(max));
 }
 const server = http.createServer((req, res) => {
-    res.end(`Your new Pokemon name is ${createName()}`)
+    res.end(`${createName()}`)
 });
 
 function createName() {
```

Comment:
Consider splitting your functionality into separate `createName()` and `createMadlib()` functions. OK for a small exercise like this, but can get nasty on larger projects.